src/tests/owners.rs

```diff
@@ -203,36 +203,26 @@ fn owners_can_remove_self() {
 */
 #[test]
 fn check_ownership_two_crates() {
-    let (_b, app, middle) = app();
+    let (app, anon, user) = TestApp::with_user();
 
-    let (krate_owned_by_team, team) = {
-        let conn = app.diesel_database.get().unwrap();
-        let u = new_user("user_foo").create_or_update(&conn).unwrap();
-        let t = new_team("team_foo").create_or_update(&conn).unwrap();
-        let krate = CrateBuilder::new("foo", u.id).expect_build(&conn);
-        add_team_to_crate(&t, &krate, &u, &conn).unwrap();
+    let (krate_owned_by_team, team) = app.db(|conn| {
+        let t = new_team("team_foo").create_or_update(conn).unwrap();
+        let krate = CrateBuilder::new("foo", user.as_model().id).expect_build(conn);
+        add_team_to_crate(&t, &krate, user.as_model(), conn).unwrap();
```

Comment:
maybe extract `user.as_model()` from here because it's used on 210 and 211, and you do that extraction in other tests and it looks nice there? not a huge deal tho